quickstart.md

```diff
@@ -78,7 +77,28 @@ This can simply be done with `nano ~/.bashrc` (`~/.zprofile` on Mac). To save pr
 
 On MacOS, run `sudo xattr -rd com.apple.quarantine $PSPDEV` to prevent Gatekeeper from blocking the toolchain.
 
-That's it, now the PSPDEV toolchain can be used to build PSP software. Below is the sample code and how to run it. For testing on real hardware, check out [psplink](https://pspdev.github.io/psplinkusb/).
+That's it, now the PSPDEV toolchain can be used to build PSP software. Check out [Coding](#coding) section below for the sample code and how to run it. For testing on real hardware, check out [psplink](https://pspdev.github.io/psplinkusb/).
+
+### Toolchain in Docker
+{: .fs-6 .fw-700 }
+
+PSPDEV has its own [Docker container](https://hub.docker.com/r/pspdev/pspdev) for the toolchain, follow the steps below to set it up.
+
+First, install the docker using the guide from [here](https://docs.docker.com/engine/install/).
```

Comment:
This sentence can be in dependencies -> docker

Reply:
Done wouter, is that right? what do you think?

Reply:
I added a commit with what I mean. Can you take a look at it and see if that's okay for you?

Reply:
Could i merge this now?